### PR TITLE
Make tests work under native shadow DOM. Fixes #638.

### LINKED ISSUE #638: Tests don't run under native shadow DOM
Under native shadow DOM, both tests fail. They should be using the Polymer.dom APIs.

### PR DESCRIPTION
**Pull request by arthurevans**

### CODE REVIEW

**Comment by samccone** [comment]
LGTM +1 @arthurevans

File: app/test/index.html [hunk 1]
```diff
@@ -23,7 +23,8 @@
   <body>
 
     <script>
-      WCT.loadSuites(['my-greeting-basic.html', 'my-list-basic.html']);
+      WCT.loadSuites(['my-greeting-basic.html', 'my-list-basic.html',
+          'my-greeting-basic.html?dom=shadow', 'my-list-basic.html?dom=shadow']);
```
Comment (robdodson) [inline app/test/index.html]: small style nit, could these be formatted as

```
WCT.loadSuites([
  'my-greeting-basic.html',
  'my-greeting-basic.html?dom=shadow'
  'my-list-basic.html',
  'my-list-basic.html?dom=shadow'
]);
```

?

**Comment by arthurevans** [comment]
Done.

**Comment by samccone** [comment]
awwww snap A+ work @arthurevans

**Pull request merged**